src/components/tl/ucp/tl_ucp.c

```diff
@@ -147,4 +148,15 @@ __attribute__((constructor)) static void tl_ucp_iface_init(void)
     ucc_tl_ucp.super.scoll.test      = ucc_tl_ucp_service_test;
     ucc_tl_ucp.super.scoll.cleanup   = ucc_tl_ucp_service_cleanup;
     ucc_tl_ucp.super.scoll.update_id = ucc_tl_ucp_service_update_id;
+    ucs_status_t status;
+    ucp_lib_attr_t params;
+    memset(&params, 0, sizeof(ucp_lib_attr_t));
+    params.field_mask = UCP_LIB_ATTR_FIELD_MAX_THREAD_LEVEL;
+    status            = ucp_lib_query(&params);
```

Comment:
is this function available in UCX 1.10? If not we need to change check for min required version of UCX